src/python/tests/appengine/handlers/cron/oss_fuzz_setup_test.py

```diff
@@ -316,6 +317,26 @@ def test_execute(self):
         'MANAGED = True\n'
         'ISSUE_VIEW_RESTRICTIONS = none\n')
 
+    job = data_types.Job.query(
+        data_types.Job.name == 'libfuzzer_asan_i386_lib3').get()
+    self.assertIsNotNone(job)
+    self.assertEqual(job.platform, 'LIB3_LINUX')
+    self.assertItemsEqual(job.templates, ['asan', 'libfuzzer'])
+    self.assertEqual(
+        job.environment_string, 'RELEASE_BUILD_BUCKET_PATH = '
+        'gs://clusterfuzz-builds-i386/lib3/lib3-address-([0-9]+).zip\n'
```

Comment:
Should lib3-address-([0-9]+).zip have i386 in there. this might be needed since if someone downloads file it shouldn't conflict with existing one and makes it easier to see which build type is it. this kind of change will be hard to do later. so @oliverchang @jonathanmetzman wdyt ?

Reply:
I'm not sure but adding this might add a lot of ugly special casing. We don't do it for AFL builds btw.

Reply:
sgtm